puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/foreman-proxy-develop-release.yaml

```diff
@@ -0,0 +1,24 @@
+- job:
+    name: foreman-proxy-develop-release
```

Comment:
Bike shedding: I thought we based the job names on the git names so should this be smart-proxy?